Screen_OLED_Test/Screen_OLED_Test.ino

```diff
@@ -0,0 +1,19 @@
+#include "OLEDScreen.h"
+
+//7.5 hours; reading library documentation and understanding
+
+void setup(void)
+{
+  initIOModule();
+}
+
+void loop(void)
+{
+  checkTime();
+  showTime();
+  timeSave();
```

Comment:
Maybe add some condition so we don't write constantly to the EEPROM, so its lifespan can be exceeded. Maybe talk with @gb02002 how he worked on the EEPROM problem and we can implement both ideas together. Using the internal EEPROM instead of the external one.